server.js

```diff
@@ -19,6 +20,7 @@ const port = 8077;
 
 const status = { SUCCESS: "SUCCESS", FAILURE: "FAILURE" };
 
+app.use(cors());
```

Comment:
do we need cors?
is it possible to whitelist the domains instead?

Please try once, that could be a better solution for production.

Reply:
Whitelisted example service only